scripts/release/rpm/Dockerfile.mariner

```diff
@@ -4,7 +4,7 @@ FROM cblmariner.azurecr.io/base/core:${tag} AS build-env
 ARG cli_version=dev
 
 RUN tdnf update -y
-RUN tdnf install -y rpm-build gcc libffi-devel python3-devel openssl-devel make diffutils patch dos2unix python3-virtualenv perl
+RUN tdnf install -y binutils file rpm-build gcc libffi-devel python3-devel openssl-devel make diffutils patch dos2unix python3-virtualenv perl
```

Comment:
**/bin/strip** is a part of binutils package which is used at %__os_install_post phase. It is required. For CentOS and Fedora this might be included in default container image or coming as a dependency at runtime.

Reply:
**+ /usr/lib/rpm/brp-strip-unneeded /bin/strip**
find: 'file': No such file or directory
find: 'file': No such file or directory
find: 'file': No such file or directory

Reply:
Could you add some comment as justification for both `binutils` and `file` so that future developers can know why they are installed? Thanks.

Reply:
Done. Updated description.